frontend/src/app/modules/registration/registration-form/registration-form.component.html

```diff
@@ -1,17 +1,17 @@
-<form [formGroup]="formGroup" action="/signup" class="reg-form" method="POST">
+<form [formGroup]="personalDetail" action="/signup" method="POST" *ngIf="step === stepRegistration.First">
     <div class="reg-block">
         <div class="input-block">
             <span class="p-float-label">
                 <input id="float-input-first-name" type="text" formControlName="firstName" name="first-name" pInputText
                     [(ngModel)]="user.firstName" [disabled]="isNotFinishedRegistration">
                 <label for="float-input-first-name">First Name</label>
             </span>
-            <div *ngIf="formGroup.controls.firstName.touched && formGroup.controls.firstName.invalid"
+            <div *ngIf="personal_step && personalDetail.controls.firstName.touched && personalDetail.controls.firstName.invalid"
                 class="validation">
-                <small *ngIf="formGroup.controls.firstName.errors.required">Сan not be empty</small>
-                <small *ngIf="formGroup.controls.firstName.value?.length < 2">Min length 2</small>
-                <small *ngIf="formGroup.controls.firstName.value?.length > 30">Max length 30</small>
-                <small *ngIf="formGroup.controls.firstName.hasError('pattern')">May letters, dashes and space</small>
+                <small *ngIf="personalDetail.controls.firstName.errors.required">Сan not be empty</small>
+                <small *ngIf="personalDetail.controls.firstName.value?.length < 2">Min length 2</small>
+                <small *ngIf="personalDetail.controls.firstName.value?.length > 30">Max length 30</small>
+                <small *ngIf="personalDetail.controls.firstName.hasError('pattern')">May letters, dashes and space</small>
```

Comment:
Please rewrite all ngIf in validation using get and set. Example in organization settings component.
Please rewrite hasError('pattern') to errors?.pattern

Reply:
Fix